src/index.ts

```diff
@@ -40,68 +41,49 @@ export class Spinner {
 
     this.frames = frames;
     this.currentSymbol = frames[0];
-    this.framesLineCountCache = new Array(frames.length).fill(-1);
   }
 
   start(tickMs = constants.DEFAULT_TICK_MS) {
+    if (this.running) throw new Error('Spinner is already running.');
+
     this.interval = setInterval(this.tick.bind(this), tickMs);
     this.running = true;
-    if (process.stdout.getWindowSize) this.terminalWidth = process.stdout.getWindowSize()[0];
     this.currentSymbol = this.frames[0];
-    this.framesLineCountCache.fill(-1);
+
+    this.tick();
```

Comment:
was this a bug before? 👀  in that it would do the first tick/render after the `tickMs` rather than immediately

Reply:
Yes 🫣, default tick speed was fast enough that it was barely noticeable.